karaf/2.4/org.ops4j.pax.logging.cfg

```diff
@@ -0,0 +1,55 @@
+################################################################################
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+################################################################################
+
+# Root logger
+log4j.rootLogger=INFO, out, osgi:*, logstash
```

Comment:
Do we really want to log to file (out) as well as Logstash? Perhaps just:

log4j.rootLogger=INFO, osgi:*, logstash

would suffice?

This also means that we can only use logstash. Can we parameterise this as well as some users might want to use stdout & capture logs in a different way? Having logstash as the default would make it difficult to develop against this image locally as you wouldn't see any logs. Logstash is really only a deployment config I would suggest.